packages/core/src/core/auth.ts

```diff
@@ -131,7 +132,10 @@ export async function startServerAndOpenBrowser(
 
   function onSuccess(data: HandlerData) {
     s.succeed(`Logged in as ${bold(data.email)}`).stop();
-    setAuthConfig(data.cookies, base);
+    PrismicSharedConfigManager.setProps({
```

Comment:
setProps is not clear, especially in a React Ecosystem "props" mean other thing. Maybe something like setProperties, setContent.

Reply:
ok